pkg/apis/apps/install/install.go

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
I think the idea is to get daemon,replica etc out of "extensions". Open to suggestion, apps is just something we came up with after ~10m of discussions, and no one really had anything better.